projects/cds/README.md

```diff
@@ -27,3 +48,39 @@ To add a SAP Context-Driven Merchandising Carousel to a page following the steps
 * Fill in the fields of the configuration window that will be displayed, including selecting a configured strategy from the 'Strategy' drop down box.
 * Click the 'Save' button.
 * The rendered merchandising carousel should now be visible in the page you're editing.
+
+## Profiletag
+
+### CMS Component
+
+To enable profiletag, a CMSFlexComponent named 'ProfileTagComponent' will need to be created in the catalog. This component will then need to be placed into the footer. The following impex accomplishes this:
+```
+$contentCatalog=electronics-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType;&componentRef;
+;;ProfileTagComponent;ProfileTag Spartacus Component;ProfileTagComponent;ProfileTagComponent;
+
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];cmsComponents(uid, $contentCV)[mode=append]
+;;FooterSlot;ProfileTagComponent
+```
+
+This will create the component in the 'Staged' catalog. To publish, run a sync, or replace 'staged' with 'Online'
+
+### Configuration
+
+Profiletag has the following configuration interface:
+```
+interface ProfileTagConfig {
+  javascriptUrl: string;
```

Comment:
After making the configs optional in the code, can we update this README and do the same?